### PR TITLE
Gutenboarding: update design screenshots

### LINKED ISSUE #44018: Gutenboarding: regenerate thumbnails
D45183-code generates Gutenboarding sites from clean starter sites, and also ensures that the previews use the starter sites as their base.

Since the content of the previews has been changed (for the better) we need to regenerate the thumbnails.

### Prerequisites

- [ ] Deploy D45183-code

### PR DESCRIPTION
**Pull request by ramonjd**
## Changes proposed in this Pull Request

We're updating screenshots now that the previews are based on clean starter sites. 

See: https://github.com/Automattic/view-design/issues/54

Should only be merged after D45183-code is deployed

## Testing instructions

Apply D45183-code

Head over to `/new` and check out the design thumbnails.

They should look like the corresponding previews.

Fixes #44018

### CODE REVIEW

**Comment by p-jackson** [comment]
Do you know how caching works for these files? I didn't notice an issue, but the urls for these images haven't changed so I'm wondering whether they need to be cache busted in some way (or maybe it's fine because the static calypso files uses an appropriate cache time)

**Comment by p-jackson** [comment]
Screenshot 2020-07-13 at 3 57 06 PM

Looks like it's cached forever. We can keep it simple and maybe just add `-v2` to the end of the filename?

**Comment by ramonjd** [comment]
Thank you for testing @p-jackson !!! 🙇‍♂️

> Actually, just noticed that the Doyle screenshot doesn't appear up to date.

🤕 I was having trouble getting the screenshot tool to work with the sandbox. I'll try again. 

Worst case: I generate it manually :) or after D45183-code is approved and merged.

> Looks like it's cached forever. We can keep it simple and maybe just add -v2 to the end of the filename? That would hopefully be enough of a clue to future devs that they'll need to do the same.

Thanks for picking that up. I was wondering why I was seeing different things on horizon/wpcalypso too.

I could probably update the generator script to append the v`n` to the file name too.

**Comment by p-jackson** [comment]
Nice, if there's a script then just adding the date or a hash would be easier than an incrementing number then I guess.

**Comment by ramonjd** [comment]
> Nice, if there's a script then just adding the date or a hash would be easier than an incrementing number then I guess.

Got a bit ahead of myself. The script doesn't update the file from which the images are linked. 🤣

File: apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-button/index.tsx [hunk 1]
```diff
@@ -2,37 +2,61 @@
...
-			</Button>
-			{ isLaunchModalVisible && <LaunchModal onClose={ handleModalClose } /> }
+			{ isSiteUnlaunched && (
+				<Button
+					aria-expanded={ isLaunchModalVisible }
+					aria-pressed={ isLaunchModalVisible }
+					aria-haspopup="menu"
+					onClick={ handleClick }
+				>
+					{ __( 'Launch site', 'full-site-editing' ) }
```
Comment (a8ci18n) [inline apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-button/index.tsx:49]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Launch site' )` *ES Score: 13*

**Comment by simison** [comment]
Not entirely sure what happened here, @claudiucelfilip did you try rebasing this PR?


> I could probably update the generator script to append the vn to the file name too.

The script could just save a date to JSON file and you could import that and append as cache-buster into the file URL? Or do it manually like @p-jackson said, and hope we can get rid of the script soon. 🤞 😄

**Comment by claudiucelfilip** [comment]
> Not entirely sure what happened here, @claudiucelfilip did you try rebasing this PR?
Don't know what happened there either. I just tried to deploy [this PR](https://github.com/Automattic/wp-calypso/pull/43987).
I saw that the [wait-calypso-live](https://app.circleci.com/pipelines/github/Automattic/wp-calypso/66883/workflows/b72fe6b4-32e3-4336-969d-ab43639504ea/jobs/806661) test was failing but it wasn't pointing to my PR.
In the end I was notified that the Canary tests for my Calypso changes ( dd2b7027 )were OK and ready to deploy.

**Comment by ramonjd** [comment]
Uh, what happened here? 🤣 

I'm going to force my local changes to fix this branch.

 This PR wasn't ready to deploy. It was waiting on requested changes from @p-jackson

**Comment by andrewserong** [comment]
Looking really good, the only screenshot where I noticed a discrepancy between the preview and the screenshot is Overton, where the "Learn more" CTA button in the screenshot has dark text on the blue button, but the preview correctly shows white on blue.

image

**Pull request merged**